data/data/bootstrap/files/usr/local/bin/kubelet.sh.template

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC1091  # using path on bootstrap machine
+. /usr/local/bin/bootstrap-service-record.sh
+
+/usr/bin/hyperkube \
+  kubelet \
+    --anonymous-auth=false \
+    --container-runtime=remote \
+    --container-runtime-endpoint=/var/run/crio/crio.sock \
+    --runtime-request-timeout="${KUBELET_RUNTIME_REQUEST_TIMEOUT}" \
+{{- if .UseIPv6ForNodeIP }}
+    --node-ip=:: \
+{{- end}}
+    --pod-manifest-path=/etc/kubernetes/manifests \
+    --minimum-container-ttl-duration=6m0s \
+    --cluster-domain=cluster.local \
+    --cgroup-driver=systemd \
+    --serialize-image-pulls=false \
+    --v=2 \
+    --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
+    --pod-infra-container-image="${MACHINE_CONFIG_INFRA_IMAGE}"
```

Comment:
I don't think adding this template is sufficient, since we're missing the systemd unit change from https://github.com/openshift/installer/pull/4742 to run this script

I guess that means we either need to backport that as well, or else add the `--node-ip` to the templated systemd unit file e.g https://github.com/openshift/installer/blob/release-4.7/data/data/bootstrap/systemd/units/kubelet.service.template